mod_reforged/hooks/config/strings.nut

```diff
@@ -935,11 +935,12 @@ foreach (vanillaDesc in vanillaDescriptions)
  	}),
 	RF_ConcussiveStrikes = ::UPD.getDescription({
  		Fluff = "A strike to the head from this character means goodnight!",
+ 		Requirement = "Mace and Blunt Damage"
  		Effects = [{
 			Type = ::UPD.EffectType.Passive,
 			Description = [
-				"Hits to the head with any weapon apply the [Dazed|Skill+dazed_effect] effect for 1 [turn|Concept.Turn]. This duration is increased to 2 [turns|Concept.Turn] for one-handed maces.",
-				"Hits to the head with two-handed maces apply the [Stunned|Skill+stunned_effect] for 1 [turn|Concept.Turn], and if the target is immune to being [stunned|Skill+stunned_effect], apply [Dazed|Skill+dazed_effect] for 1 [turn|Concept.Turn]."
+				"Hits to the head with one-handed maces apply the [Dazed|Skill+dazed_effect] effect for 1 [turn|Concept.Turn] and if the target is already [dazed|Skill+dazed_effect], apply [Stunned|Skill+stunned_effect] for one [turn|Concept.Turn].",
+				"Hits to the head with two-handed maces apply the [Stunned|Skill+stunned_effect] for 1 [turn|Concept.Turn].",
```

Comment:
Is this nerf to two-handed weapons intentional? Now this is a dead perk for two-handed weapons if the enemy is stun immune. Before you would still apply dazed at the very least. 

One-handed weapons on the other hand can inflict dazed against stun-immune enemies just fine as before

Reply:
The original tooltip was misleading. Two-Handed maces already apply Dazed by default via the Cudgel skill, even against stun immune enemies. However, they won't apply it against Daze immune enemies. 

So nothing has been changed for 2-handed maces in this perk.

However, we could consider buffing it for 2-handed maces so that Daze-immune enemies still get Dazed applied to them?